test/rendering.test.js

```diff
@@ -458,7 +458,7 @@ describe('Rendering', () => {
         'access-control-allow-origin': '*',
         'content-type': 'text/html; charset=utf-8',
         'last-modified': 'Fri, 30 Apr 2021 03:47:18 GMT',
-        'x-surrogate-key': 'zxdhoulVcSRWb0Ky ZHQXDa0L7jSHQHPX foo-id_metadata super-test--helix-pages--adobe_head',
+        'x-surrogate-key': 'zxdhoulVcSRWb0Ky SUhNxkR2spoxY489 foo-id_metadata super-test--helix-pages--adobe_head',
```

Comment:
there was an error in the path computation for foldermapped paths... now it takes the path of the new target, which is correct